.github/workflows/release-cli.yml

```diff
@@ -86,9 +91,140 @@ jobs:
           name: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
           path: bin/*
 
+  package-deb:
+    name: Build deb package
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: amd64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: arm64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: bin
+          pattern: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
+          merge-multiple: true
+
+      - name: create dir structure for .deb package
+        run: |
+          mkdir -p .debpkg/usr/bin
+
+          # when we download from deb, the binary should be named kubetail
+          mv 'bin/kubetail-${{ matrix.GOOS }}-${{ matrix.GOARCH }}' bin/kubetail 
```

Comment:
You can just copy from `bin/` directly into `.depbkg/usr/bin` and change the name in one step.